code/modules/flufftext/Hallucination.dm

```diff
@@ -692,6 +692,7 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 		feedback_details += "Type: Talk, Source: [person.real_name], Message: [message]"
 		to_chat(target, message)
 		if(target.client)
+			animate_chat(person, message, understood_language, null, list(target.client), 50)
 			target.client.images |= speech_overlay
```

Comment:
1. Shouldn't we add `INVOKE_ASYNC` for `animate_chat`?
2. For maximal realism, we can use `animate_speechbubble` instead of `client.images`; see `/proc/send_message`

Reply:
yes